lib/stitch_plan/stitch_plan.py

```diff
@@ -146,7 +146,7 @@ def __json__(self):
                     num_trims=self.num_trims,
                     num_stitches=self.num_stitches,
                     bounding_box=self.bounding_box,
-                    estimated_thread=self.estimated_thread
+                    estimated_thread=self.estimated_thread,
```

Comment:
This change can probably go away now too.

Reply:
Removed in latest commit.